ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/nodes/pairs/values/TomlValue.kt

```diff
@@ -10,9 +10,10 @@ import com.akuleshov7.ktoml.writers.TomlEmitter
 
 /**
  * Base class for all nodes that represent values
- * @property lineNo - line number of original file
  */
-public sealed class TomlValue(public val lineNo: Int) {
+public sealed class TomlValue {
+    @Deprecated(message = "lineNo is deprecated. Will be removed in next releases.")
```

Comment:
I will argue on this. I still think that `lineNo` is very important on the side of nodes. 
For debug purposes and for the future usage. In good AST they always store it inside nodes... Actually this is the only real concern in this PR for me.

@NightEule5 why you do not like line number here so much? :D

Reply:
Can I isolate this change in a separate PR?

Reply:
after looking more into the code, I understood your pain. 
Yes, we can really remove lineNo from TomlKey and TomlValue: but anyway we should keep it on KeyValue and Table level, it's very important...

Reply:
Sure, we can add it back. I just thought it's only used in the parsing process